modules/@apostrophecms/i18n/i18n/fr.json

```diff
@@ -0,0 +1,406 @@
+  "leavePageHeading": "Quitter {{ oldTitle }} pour éditer {{ newTitle }}?",
+  "listJoiner": ", ",
+  "live": "En Live",
+  "lastUpdatedBy": "Enregistré dernièrement le {{ updatedAt }} <br /> par {{ updatedBy }}",
+  "loadDocFailed": "Le document demandé n'a pas été trouvé.",
+  "locale": "Langue",
+  "localize": "Changer la langue",
+  "localizeContent": "Changer la langue contenu",
+  "localized": "{{ type }} {{ title }} changé en {{ locale }}",
+  "localizeAllRelated": "Changer la langue de tous les documents associés et écraser les documents existants",
+  "localizeNewRelated": "Changer la langue des nouveaux documents associés",
+  "localizingBusy": "Changement de la langue du contenu",
+  "login": "Connexion",
+  "loginErrorGeneric": "Une erreur est survenue. Veuillez réessayer.",
+  "loginPageBothRequired": "Le nom d'utilisateur ainsi que le mot de passe sont requis.",
+  "loginPageBadCredentials": "Vos identifiants sont incorrects, ou il n'existe pas de tel utilisateur",
+  "loginPageTitle": "Connexion",
+  "loginRequired": "Connexion requise",
+  "logOut": "Déconnexion",
+  "logOutNotLoggedIn": "Vous n'étiez pas connecté.",
+  "manageDocType": "Gérer {{ type }}",
+  "manageDraftSubmissions": "Gérer les soumissions de brouillon",
+  "managePages": "Gérer les pages",
+  "loginMaxAttemptsReached": "Trop de tentatives. Vous pouvez ré-essayer dans une minute.",
+  "loginMaxAttemptsReached_plural": "Trop de tentatives. Vous pouvez ré-essayer dans {{ count }} minutes.",
+  "maxLabel": "Max :",
+  "maxUi": "Max : {{ number }}",
+  "minSize": "Taille minimale de {{ width }}x{{ height }}",
+  "mediaCreatedDate": "Téléversé : {{ createdDate }}",
+  "mediaDimensions": "Dimensions : {{ width }} 𝗑 {{ height }}",
+  "mediaFileSize": "Taille du fichier : {{ fileSize }}",
+  "mediaKB": "{{ size }} Ko",
+  "mediaMB": "{{ size }} Mo",
+  "mediaUploadViaDrop": "Vous pouvez les lâcher",
+  "mediaUploadViaExplorer": "Ou cliquez pour ouvrir l'explorateur de fichiers",
+  "minLabel": "Min :",
+  "minUi": "Min : {{ number }}",
+  "modify": "Modifier",
+  "modifyOrDelete": "Modifier / Supprimer",
+  "moreOptions": "Plus d'options",
+  "moreOperations": "Plus d'operations",
+  "multipleEditors": "Editeurs multiples",
+  "newDocType": "Nouveau: {{ type }}",
+  "newItem": "Nouvel élément",
+  "newPage": "Nouvelle page",
+  "newSearch": "Nouvelle recherche",
+  "next": "Suivant",
+  "nextPage": "Page suivante",
+  "no": "Non",
+  "noDraftSubmissions": "Pas de soumission de brouillon à gérer",
+  "noItemsSelected": "Pas d'élément sélectionné",
+  "noJustSwitchLocales": "Non, changer seulement de paramètre régional",
+  "noLongerPublished": "N'est plus publié.",
+  "noMediaFound": "Pas de média trouvé",
+  "noNewRelatedDocuments": "Bien que ce document ait des documents associés, aucun d'entre eux n'est nouveau pour les langues que vous avez sélectionnés.",
```

Comment:
🤔 I didn't exactly understand the meaning of `none of them are new to the locales you have selected.` or `ninguno de ellos es nuevo para las configuraciones regionales que ha seleccionado`...

In what way aren't they "new" to the selected locales?

Reply:
If you choose "localize new related documents," as opposed to "localize new related documents and overwrite existing documents," and there are related documents but all of them already exist in the other locales, you may see this message.

Reply:
Ok, the translation kinda makes sense then!